ntoskrnl/cc/view.c

```diff
@@ -646,6 +646,10 @@ CcRosLookupVacb (
                            VACB_MAPPING_GRANULARITY,
                            FileOffset))
         {
+            /* HACK ? */
+            if (CcRosVacbGetRefCount(current) == 0)
```

Comment:
This is clearly not a proper fix. It's just hiding a nasty bug. I'm ok with hacks to make the system more stable, but this case doesn't seem to justify this. It's reproducable, which makes it a very good case to actually debug and fix the issue, rather than hacking it away.

Reply:
I do trust Timos judgement here ofc.

Reply:
The hack was removed.